src/FileHandler.cpp

```diff
@@ -1,7 +1,7 @@
 #include "../include/FileHandler.hpp"
 #include "../include/WebServer.hpp"
 
-static std::string getFilePath(std::string relativePath)
+std::string FileHandler::getFilePath(std::string relativePath)
```

Comment:
why is this not a static function anymore?

Reply:
oh, you need it in the request handler

Reply:
As I said above in the PR comment, I used it in the path validation, so I changed it to public so I could use it.